ang/civicase/case/actions/directives/case-actions.directive.js

```diff
@@ -310,11 +310,19 @@
           },
 
           gotoWebform: function (selectedCase, path, clientId) {
-            clientId = 'cid' + clientId;
-            var url = CRM.url(path, {
-              case1: selectedCase.id,
-              [clientId]: selectedCase.client[0].contact_id
-            });
+            var url;
```

Comment:
Also this function lacks documentation. And unit test. Even if we dont want to add unit test now, documentation must be added.